pkg/testdata/models/apis/eks/0000-00-00/generator.yaml

```diff
@@ -1,3 +1,10 @@
+ignore:
+  field_paths:
+    # Test nested field path of multiple levels
+    - Nodegroup.Version
+    - CreateNodegroupInput.Version
+    - CreateNodegroupInput.ScalingConfig.DesiredSize
```

Comment:
Quoting from the docstring:
```
// Set of field paths to ignore. The name here should be the original name of
// the field as it appears in AWS SDK objects. You can refer to a field by
// giving its "<shape_name>.<field_name>". For example, "CreateApiInput.Name".
```

Would it be possible to use `ScalingConfig.DesiredSize` (assuming `ScalingConfig` is the name of the type) to achieve what you'd like?

Reply:
Yeah, that works :facepalm: Thanks! :D

I was trying to help @ikarldasan on slack to get progress on https://github.com/crossplane/provider-aws/pull/755 - but I didn't know how the generator config worked and dug into this hole.

So for that case, to ignore TemplateName, it would simply be:

```yaml
ignore:
  field_paths:
    - Template.TemplateName
```

I though that was tested but apparently not :)